Workbooks/UsageMetrics/Usage - Events/Usage - Events.workbook

```diff
@@ -1010,7 +1010,7 @@
                   "id": "732d8494-7a52-426a-a963-27977fe443d0",
                   "version": "KqlParameterItem/1.0",
                   "name": "Properties",
-                  "label": "PROPERTIES ⚙️",
```

Comment:
Just curios, why are we changing the emoji?

Reply:
It's better to use this emoji for performance configuration.